_data/soulhunter10737/soulhunter10737.md

```diff
@@ -0,0 +1,7 @@
+---
+name: SHIBAM DHAR
+institution: KCCITM
+profile_pic:soulhunter10737.jpj
+quote:Brightness can only be seen in the presemce of darkness.
```

Comment:
Add a space between the `:` and your text.

Reply:
Done